app/src/main/java/com/asterisk/tuandao/themoviedb/ui/home/HomeAdapter.kt

```diff
@@ -8,34 +8,32 @@ import com.asterisk.tuandao.themoviedb.R
 import com.asterisk.tuandao.themoviedb.data.source.model.Movie
 import com.asterisk.tuandao.themoviedb.databinding.ItemHomeMovieBinding
 
-class HomeAdapter(private var movies: List<Movie>, private val homeViewModel: HomeViewModel) :
-    RecyclerView.Adapter<HomeAdapter.HomeViewHolder>() {
+class HomeAdapter(private var movies: List<Movie>, private val viewModel: HomeViewModel) :
+        RecyclerView.Adapter<HomeAdapter.HomeViewHolder>() {
```

Comment:
If we use inheritance, then the superclass constructor call or list of implemented interfaces should be located on the same line as the parenthesis:
https://kotlinlang.org/docs/reference/coding-conventions.html#formatting

Reply:
cái này dài vượt quá quy định rồi nên mình xuống dòng